code/datums/components/storage/concrete/pockets.dm

```diff
@@ -50,7 +50,7 @@
 	if(istype(I,/obj/item/clothing/head/mob_holder))
 		var/obj/item/clothing/head/mob_holder/mausholder = I
 		if(locate(/mob/living/simple_animal/mouse) in mausholder.contents)
-			return TRUE
+			return .
```

Comment:
```suggestion
			return
```

Reply:
shouldn't this be false?

Reply:
Does "return" with no "." after it return the value of ".", or just FALSE?

Reply:
It returns the value of `.`
Check the reference:
http://www.byond.com/docs/ref/index.html#/proc/var/.
http://www.byond.com/docs/ref/index.html#/proc/return